src/genotet.js

```diff
@@ -18,6 +18,64 @@ var genotet = {};
 genotet.test = function() {};
 
 /**
+ * Network extern.
+ */
+genotet.network = {};
+
+/**
+ * Expression extern.
+ */
+genotet.expression = {};
+
+/**
+ * Binding extern.
+ */
+genotet.binding = {};
+
+/**
+ * Bed extern.
+ */
+genotet.bed = {};
+
+/**
+ * Mapping extern.
+ */
+genotet.mapping = {};
+
+/** @enum {string} */
+genotet.FileType = {
+  NETWORK: 'network',
+  EXPRESSION: 'expression',
+  BINDING: 'binding',
+  BED: 'bed',
+  MAPPING: 'mapping'
+};
+
+/** @enum {string} */
+genotet.ViewType = {
+  NETWORK: 'network',
+  EXPRESSION: 'expression',
+  BINDING: 'binding'
+};
+
+/** @enum {string} */
+genotet.ListQueryType = {
```

Comment:
Since listing is all carried out in genotet.data, it'd better to be defined under
```js
genotet.data.ListQueryType
```